File: source/utility/print_help.go
package utility

import (
	"fmt"
)

// Print Help Command Output
func PrintHelp() {
	fmt.Println("Hyaena Technologies: Data Interchange Server")
	fmt.Println("")
	fmt.Println("")
	fmt.Println("Commands:							Description:")
	fmt.Println("")
	fmt.Println("configuration 						Configure Server with server-configuration.yaml")
	fmt.Println("exit 								Exit Server")
	fmt.Println("help								Print List of Commands and Flags")
	fmt.Println("serve 								Serve Web Applcation")
	fmt.Println("version							Print Version Number")
	fmt.Println("")
	fmt.Println("")
	fmt.Println("Flags:				    			Description:")
	fmt.Println("")
	fmt.Println("--config. -config					Configure Server with server-configuration.yaml")
	fmt.Println("--exit, -exit, --e, -e				Exit Server")
	fmt.Println("--help, -help, --h, -h				Print List of Commands and Flags")
	fmt.Println("--version, -version, --v, -v		Print Version Number")
}

File: src/utility/print_help.rs
// Print Help Command Output
fn PrintHelp() {
	Println!("Hyaena Technologies: Data Interchange Server");
	Println!("");
	Println!("");
	Println!("Commands:							Description:");
	Println!("");
	Println!("configuration 						Configure Server with server-configuration.yaml");
	Println!("exit 								Exit Server");
	Println!("help								Print List of Commands and Flags");
	Println!("serve 								Serve Web Applcation");
	Println!("version							Print Version Number");
	Println!("");
	Println!("");
	Println!("Flags:				    			Description:");
	Println!("");
	Println!("--config. -config					Configure Server with server-configuration.yaml");
	Println!("--exit, -exit, --e, -e				Exit Server");
	Println!("--help, -help, --h, -h				Print List of Commands and Flags");
	Println!("--version, -version, --v, -v		Print Version Number");
}
